RPGManager_development/src/rpg_database/character_sheet/Fields.java

```diff
@@ -2,7 +2,8 @@
 
 public enum Fields {
 	NAME(String.class), AGE(Integer.class), XP(Integer.class), GENDER(Gender.class), SPEED(Integer.class), BACKGROUND(Background.class),
-	BASECLASS(BaseClasses.class), SPECIALIZATIONCLASS(SpecializationClasses.class), MONEY(Integer.class);
+	BASECLASS(BaseClasses.class), SPECIALIZATIONCLASS(SpecializationClasses.class), MONEY(Integer.class), LEVEL(Integer.class),
+	DEFENSE(Integer.class), ARMOR(Integer.class), HEALTH(Integer.class), MANA(Integer.class);

```

Comment:
- Az ARMOR helyett lehetne ARMOR_RATING mert gondolj bele hogy lesz Armor Type is később
- HEALTH is lehetne HEALTH_POINTS, vagy HP, szerintem érthetőbb
- Ezt a konvenciót követve viszont a MANA is lehetne MANA_POINTS vagy MP de erre nem foglak kötelezni